HelpSource/scdoc.js

```diff
@@ -513,6 +513,9 @@ function fixTOC() {
         }, false);
         toc.insertBefore(close_link,toc.firstChild);
         resize_handler();
+        if(storage.tocOpen == "yes") {
```

Comment:
space missing between `if` and `(`

`===` is preferred over `==` in javascript

Reply:
done!